tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml.cs

```diff
@@ -1,22 +1,40 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using AdaptiveCards.Rendering.WinUI3;
-using CommunityToolkit.Mvvm.Messaging;
-using DevHome.SetupFlow.Models.Environments;
+using CommunityToolkit.WinUI;
+using DevHome.Common.Extensions;
 using DevHome.SetupFlow.ViewModels;
-using DevHome.SetupFlow.Windows;
+using Microsoft.UI.Dispatching;
 using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Controls;
+using Windows.UI.ViewManagement;
 
 namespace DevHome.SetupFlow.Views;
 
 public sealed partial class SummaryView : UserControl
 {
+    private readonly UISettings _uiSettings = new();
+
+    private const double BaseWidth = 550;
+
     public SummaryView()
     {
         this.InitializeComponent();
+
+        var textScale = _uiSettings.TextScaleFactor;
+        _uiSettings.TextScaleFactorChanged += HandleTextScaleFactorChanged;
```

Comment:
I may be skewed since I was just looking at code where event subscriptions were keeping object alive, but consider unsubscribing in the Unloaded event (or possibly look at memory in the VS performance profiler to be sure this is getting cleaned up)

Reply:
Added an unload event.